### PR TITLE
05 map points

### PR DESCRIPTION
**Pull request by llaulier**
API context, Marker added on map, with Icon

### CODE REVIEW

File: src/contexts/ApiContext.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  }, []);
+
+  const getCapsules = () => {
+    axios
+      .get('http://localhost:4242/capsules')
+      .then((res) => {
+        setCapsules(res.data);
+      })
+      .catch((err) => {
+        console.log(err);
```
Comment (RoxaneDrelon) [inline src/contexts/ApiContext.jsx]: API call for poi done with .env but not the capsules one

**Review by RoxaneDrelon — CHANGES_REQUESTED** [review]
Streamline API calls

File: src/contexts/ApiContext.jsx (hunk 1 above)
Comment (llaulier) [reply src/contexts/ApiContext.jsx]: Oh... right !

**Review by RoxaneDrelon — DISMISSED** [review]
Seems good

File: src/components/map/Map.jsx [hunk 2]
```diff
@@ -1,10 +1,22 @@
...
+  iconSize: [34, 34],
+});
 
 const Leaflet = () => {
   const value = useContext(LocationContext);
+  const { poi } = useContext(ApiContext);
+
+  const allPoi = [...poi];
+
+  console.log(IconStyle);
```
Comment (LoicBrassart) [inline src/components/map/Map.jsx]: No console.log!

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Minor Evil

**Review by LoicBrassart — APPROVED** [review]
SnE

**Review by RoxaneDrelon — APPROVED** [review]

**Pull request merged**